docs/source/ops.rst

```diff
@@ -10,49 +10,98 @@ Operators
 .. note::
   All operators have native support for TorchScript.
 
+
+Detection Operators
+~~~~~~~~~~~~~~~~~~~
+
 .. autosummary::
     :toctree: generated/
     :template: function.rst
 
     batched_nms
+    nms
+    roi_align
+    roi_pool
+
+.. autosummary::
+    :toctree: generated/
+    :template: class.rst
+
+    FeaturePyramidNetwork
+    MultiScaleRoIAlign
+    RoIAlign
+    RoIPool
+
+
+Box Operators
+~~~~~~~~~~~~~
```

Comment:
Maybe add one line description for each?

Reply:
Sure, as long as we have something meaningful and helpful to say that doesn't repeat what we say on the summary above.